swift_browser_ui/ui/middlewares.py

```diff
@@ -27,6 +29,21 @@ def return_error_response(error_code: int) -> web.Response:
         )
 
 
+@web.middleware
+async def check_session_at(
+    request: web.Request,
+    handler: AiohttpHandler,
+) -> web.Response:
+    """Raise on expired sessions."""
+    session = await aiohttp_session.get_session(request)
+    if "at" in session:
+        if session["at"] + 28800 < time.time():
```

Comment:
```suggestion
    if "at" in session and session["at"] + 28800 < time.time():
```
I don't see any reason not do it like this also, or am I missing something ?

Reply:
Nope, that's fine – the comparison aborts if in fails so it won't generate a KeyError. I'll commit it 👍🏼

Reply:
maybe the indentation should have been fixed (my bad), but we can amend the last commit